gensim/test/test_utils.py

```diff
@@ -84,6 +85,48 @@ def test_decode_entities(self):
         expected = u'It\x92s the Year of the Horse. YES VIN DIESEL \U0001f64c \U0001f4af'
         self.assertEqual(utils.decode_htmlentities(body), expected)
 
+    def test_open_file_existent_file(self):
```

Comment:
No need to check that this doesn't raise an exception if something goes wrong - unittest fail the current test and will show needed information (here and everywhere). Sorry, apparently I misunderstood you in the previous question